app/logic/Mail/EWS/EWSAccount.ts

```diff
@@ -47,9 +47,11 @@ export class EWSAccount extends MailAccount {
     return this.authMethod != AuthMethod.OAuth2 || this.oAuth2?.isLoggedIn;
   }
 
-  async login(interactive: boolean): Promise<void> {
-    await ensureLicensed();
-    await super.login(interactive);
+  async verifyLogin(): Promise<void> {
+    await this.commonLogin(true);
+  }
+
+  async commonLogin(interactive): Promise<void> {
```

Comment:
```suggestion
  protected async loginCommon(interactive): Promise<void> {
```
(same elsewhere)

Reply:
Can you please also change the function name?

Reply:
Sorry, I overlooked that you wanted the function name changed.